src/components/login_page/login_page.js

```diff
@@ -1,12 +1,145 @@
 import React from "react";
+import axios from 'axios';
+import Container from 'react-bootstrap/Container';
+import "./login_page.css";
+import user_image from "./assets/img/user.svg"
+import RegisterPage from "../register_page/register_page";
 
+import {
+    BrowserRouter as Router,
+    Routes,
+    Route,
+    Link
+} from "react-router-dom";
 
-export class LoginPage extends React.Component{
-    render() {
-        return (
-            <div>
-                <h1>Login Page</h1>
-            </div>
-        )
+
+const LoginPage = () => {
+    const [formValue, setFormValue] = React.useState({
+        email: "",
+        password: ""
+    });
+
+    const handleSubmit = async (event) => {
+        const loginFormData = new FormData();
+        loginFormData.append("email", formValue.email)
+        loginFormData.append("password", formValue.password)
+        event.preventDefault()
+
+        const keyArray = ["email", "password"]
+        for (const currentKey of keyArray){
+            let currentElement = document.querySelector(`[name='${currentKey}']`);
+            let currentErrorMessage = currentElement.parentElement.querySelector("p");
+            currentElement.classList.remove("validation_error");
+            currentErrorMessage.classList.remove("error_message_shown");
+        }
+
+        try {
+            const response = await axios({
+                method: "post",
+                url: "http://3.65.253.196/auth/jwt/create/",
```

Comment:
Please, change to "https://g6bcybbjx1.execute-api.eu-central-1.amazonaws.com/auth/jwt/create/"